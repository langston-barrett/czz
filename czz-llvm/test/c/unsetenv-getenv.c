#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]) {
  int success = unsetenv("FOO");
  printf("success=%d\n", success);
  printf("FOO=%d\n", getenv("FOO") == NULL);
  return 0;
}
