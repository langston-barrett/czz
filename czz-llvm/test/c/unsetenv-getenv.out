success=0
FOO=1
